sympy/functions/combinatorial/tests/test_comb_factorials.py

```diff
@@ -471,6 +471,89 @@ def test_binomial():
     assert expand_func(binomial(n, -2, evaluate=False)) == 0
     assert expand_func(binomial(n, k)) == binomial(n, k)
 
+def test_multinomial():
```

Comment:
The tests are excessive for the code that was added. Just use symbolic values and check that the symbolic result is correct, then a couple of numeric cases. Any test with symbols that reduces to a number is only testing the already tested gamma functions.

Reply:
Okay sir.

Reply:
Sir, actually @czgdp1807 had asked me to add more tests for increase in code coverage.

Reply:
Tests should have been added in such  a way that uncovered is tested. Adding more tests doesn't necessarily mean that the coverage will increase.